src/bo4e/validators.py

```diff
@@ -72,3 +73,46 @@ def check_bis_is_later_than_von(instance: _VonBisType, attribute, value):
 OBIS_PATTERN = r"((1)-((?:[0-5]?[0-9])|(?:6[0-5])):((?:[1-8]|99))\.((?:6|8|9|29))\.([0-9]{1,2})|(7)-((?:[0-5]?[0-9])|(?:6[0-5])):(.{1,2})\.(.{1,2})\.([0-9]{1,2}))"
 #: an attr validator
 obis_validator = attr.validators.matches_re(OBIS_PATTERN)
+
```

Comment:
Übersehe ich etwas oder ist die Malo Validierung hier nur zufällig im PR gelandet? 
Wird die irgendwo schon genutzt und getestet?

Reply:
die malo-validierung wird tatsächlich genutzt. aber wo war sie vorher? und wie kann das funkjtionieren? :D

Reply:
bd41a862e7c31a81a735246cf677ed180e7faa27 ok, jetzt ist sie weg. keine ahnung, was da in meinem kopf vorging